.devcontainer/postcreate.sh

```diff
@@ -1,4 +1,6 @@
-npm install -g @devcontainers/cli
+#!/bin/bash
+
+go install github.com/swaggo/swag/cmd/swag@v1.16.3
 
 go get -u google.golang.org/protobuf/cmd/protoc-gen-go
```

Comment:
You can remove the proto-gen installation, that's a relict from our GRPC api days.

Reply:
rebased and removed proto-gen